services/_shared/personas/publish_check.py

```
@@ -0,0 +1,169 @@
+            pr_number=pr_number,
+            head_sha=head_sha,
+            conclusion=conclusion,
+            summary=title,
+            findings_count=findings_count,
+            blocking=blocking,
+            degraded_reason=(
+                degraded_reason or ("check_publish_failed" if publish_failed else None)
+            ),
+        )
+    except Exception as e:  # noqa: BLE001 — best-effort per this module's own contract
+        log.error(
```

Ruff (0.15.20)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)